src/main/java/io/reactivex/internal/util/QueueDrainHelper.java

```diff
@@ -359,4 +360,29 @@
         
         return false;
     }
+    
+    /**
+     * Creates a queue: spsc-array if capacityHint is positive and 
+     * spsc-linked-array if capacityHint is negative; in both cases, the
+     * capacity is the absolute value of prefetch.
+     * @param <T> the value type of the queue
+     * @param capacityHint the capacity hint
+     * @return the queue instance
+     */
+    public static <T> Queue<T> createQueue(int capacityHint) {
+        if (capacityHint < 0) {
```

Comment:
In my opinion it makes reading the code a bit easier and 3 lines will be saved too but yeah it's just a nit.